clippy_lints/src/copies.rs

```diff
@@ -225,9 +225,11 @@ fn if_sequence(mut expr: &Expr) -> (SmallVector<&Expr>, SmallVector<&Block>) {
     let mut conds = SmallVector::new();
     let mut blocks = SmallVector::new();
 
-    while let ExprIf(ref cond, ref then_block, ref else_expr) = expr.node {
+    while let ExprIf(ref cond, ref then_expr, ref else_expr) = expr.node {
         conds.push(&**cond);
-        blocks.push(&**then_block);
+        //FIXME
+        //blocks.push(&**then_expr);
```

Comment:
You need to change the return type to Expr instead of block. Then it'll work again.

Not sure about the use sites oft this function. Might need to match on the kind